package.json

```diff
@@ -17,7 +17,8 @@
     "megs"
   ],
   "scripts": {
-    "pretest": "jshint --config ./node_modules/yui-lint/jshint.json ./index.js",
+    "lint": "eslint index.js",
```

Comment:
Make this `eslint --fix .` that way it lints the tests and fixes any linting issues that it knows about. It will prevent issues later on :)